src/templates/main/components/predictions-list.html

```diff
@@ -36,7 +36,7 @@
       <p class="mb-1 model-id">{{ prediction.id }}</p>
     </div>
     <div class="col-1" style="padding-left: 0;">
-      {% with image_path='img/plangs/'|add:prediction.model.implementation_language|lower|add:'.svg' %}
+      {% with image_path='img/plangs/'|add:prediction.model.implementation_language.svg_path %}
```

Comment:
Now is certain that this path `static/img/plangs/....svg`  contains a svg path attached to it